src/dataflow/Cargo.toml

```diff
@@ -45,6 +45,7 @@ serde = { version = "1.0.124", features = ["derive"] }
 serde_json = "1.0.64"
 timely = { git = "https://github.com/TimelyDataflow/timely-dataflow", default-features = false, features = ["bincode"] }
 tokio = { version = "1.4.0", features = ["fs", "rt", "sync" ] }
-tokio-util = { version = "0.6.5", features = ["codec"] }
+tokio-util = { version = "0.6.4", features = ["codec"] }
+pubnub-hyper = { git = "https://github.com/petrosagg/pubnub-rust", branch = "tokio-v1" }
```

Comment:
Aw, shoot, too bad about having to fork. In the interest of reproducible builds in the future, can you create a MaterializeInc-owned fork, and merge your Tokio v1.0 upgrade into `master`? Feature branches on personal forks have a way of disappearing, and then it's impossible to build that version of Materialize in the future.

Reply:
My plan was to put the PR out for review in a runnable state (hence the personal fork) and merge once my upstream PR https://github.com/pubnub/rust/pull/61 is merged and released.

I'll give it a few days and implement your proposal if we're still blocked

Reply:
You are more optimistic than I! Most crates are on a release cadence of several months, if not years. :) Sounds good, though.

Reply:
I'm too excited to wait. Created the mz-owned fork and will merge as soon as tests pass!